src/CodeExplorerBundle/Resources/config/services.yml

```diff
@@ -1,9 +1,9 @@
 services:
+    _defaults: { public: false }
+
     code_explorer.twig.source_code_extension:
-        public:    false
-        class:     CodeExplorerBundle\Twig\SourceCodeExtension
-        tags:
-            - { name: twig.extension }
+        class: CodeExplorerBundle\Twig\SourceCodeExtension
+        tags: [twig.extension]
```

Comment:
Sure, if there is a consensus on this, I'd be pleased to add autoconfigure and classnamed services here, as explained in the description (I just want to gather opinions here first) :)